components/Navbar.vue

```diff
@@ -326,11 +319,14 @@ export default class NavbarMenu extends mixins(
     clearSession()
   }
 
+  get isMobileMenuOpen() {
+    const navbarMenu = document.querySelector('.navbar-menu')
+    return navbarMenu?.classList.contains('is-active')
+  }
```

Comment:
The function seems not related to `mobile`. How about naming `isNavbarMenuOpen`?

Reply:
it is related to mobile
the class is-active appears only when mobile menu is opened